contract_api/consumers/service_event_consumer.py

```diff
@@ -112,6 +112,62 @@ def _get_new_assets_url(self, org_id, service_id, new_ipfs_data):
                                                                  service_id)
         return assets_url_mapping
 
+    def create_service_media(self,org_id,service_id,service_media,service_row_id,assets_url):
+        count = 0;
+        if(service_media!=None):
```

Comment:
No need for braces here. Please use consistent coding style

Reply:
Also use "is not" instead of !=

Reply:
http://jaredgrubb.blogspot.com/2009/04/python-is-none-vs-none.html

Reply:
These are addressed